frontend/src/ui/OnetLink.stories.tsx

```diff
@@ -0,0 +1,14 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0';
+import React from 'react';
+import OnetLink, { OnetLinkProps } from './OnetLink';
+
+export default {
+  title: 'JobHopper/OnetLink',
+  component: OnetLink,
+} as Meta;
+
+export const OnetLink_Computer_Programmer: Story<OnetLinkProps> = args => (
```

Comment:
Handles this comment: 
Please use camelCase here and name it after the occupation (computer programmer) rather than the code to improve readability.

Reply:
nit: remove underscores -- in camelCase capitalization denotes space